SoftwareName_Engine/Query/ExampleQueryMethod.cs

```diff
@@ -30,7 +30,7 @@
 
 namespace BH.Engine.Adapters.SoftwareName
 {
-    public static partial class Create
+    public static partial class Query
     {
         /***************************************************/
         /**** Public Methods                            ****/
```

Comment:
Cant comment on the line below, but might as well fix the method name, and making it extension while you are on this (as you noted yesterday 👍 )

basically, change from:

`public static string ExampleCreateMethod(ExampleObject exampleObject)`

to

`public static string ExampleQueryMethod(this ExampleObject exampleObject)`

Reply:
Oh you're right thanks! 
(P.S. you can comment on any line by clicking the blue down arrow at the bottom-left corner of the visible code)